src/Transport/Topology/MetaModel/NamespaceInfo.cs

```diff
@@ -1,45 +1,36 @@
 ﻿namespace NServiceBus.AzureServiceBus
 {
-    public class NamespaceInfo
+    using System;
+
+    // NOTE: Can namespaces switch mode?
```

Comment:
Ok, but why does project compile anyway?
I converted NamespaceInfo into a plain "value object", making `Mode` property readonly and everything works well (project build and tests). Does someone change namespace mode?